src/frontend/komponenter/HeaderMedSøk/FagsakDeltagerSøk.tsx

```diff
@@ -18,10 +18,30 @@ import { idnr } from '@navikt/fnrvalidator';
 
 import OpprettFagsakModal from './OpprettFagsakModal';
 import { useAppContext } from '../../context/AppContext';
-import IkkeTilgang from '../../ikoner/IkkeTilgang';
-import type { IFagsakDeltager, ISøkParam } from '../../typer/fagsakdeltager';
-import { fagsakdeltagerRoller } from '../../typer/fagsakdeltager';
+import {
+    FagsakDeltagerRolle,
+    fagsakdeltagerRoller,
+    type IFagsakDeltager,
+    type ISøkParam,
+} from '../../typer/fagsakdeltager';
+import { Adressebeskyttelsegradering } from '../../typer/person';
 import { obfuskerFagsakDeltager } from '../../utils/obfuskerData';
+import { PersonIkon } from '../PersonIkon';
+
+function mapFagsakDeltagerTilIkon(fagsakDeltager: IFagsakDeltager): React.ReactNode {
+    return (
+        <PersonIkon
+            kjønn={fagsakDeltager.kjønn || kjønnType.UKJENT}
+            erBarn={fagsakDeltager.rolle === FagsakDeltagerRolle.Barn}
+            adresseBeskyttelse={
+                fagsakDeltager.adressebeskyttelseGradering !== null &&
```

Comment:
Blir det mer korrekt å sjekke for `undefined` her? IntelliJ sier i det minste at den kan være `undefined` 😅 

```suggestion
                fagsakDeltager.adressebeskyttelseGradering !== undefined &&
```

<img width="908" height="464" alt="Image" src="https://github.com/user-attachments/assets/635cea59-1947-4d68-94e4-c2db4ae64715" />

Reply:
Jeg beholder begge, ettersom verdien også kan være null. Så de fleste vil oppdateres til å både sjekke på undefined og null.